get_rapid_orthophotos_for_project.py

```diff
@@ -35,27 +35,43 @@
         #These are stored in a list in an arbitrary order.
         for file in orthophoto_files:
             if file.type == 'geoPackage':
+                response=None
                 try:
-                    #Get geopackage as a bytestring
-                    geo_package_string = rapid_orthophoto_files_api.get_rapid_orthophoto_geo_package(orthophoto.id, file.id)
-                    geo_package = eval(geo_package_string)
+                    response = rapid_orthophoto_files_api.get_rapid_orthophoto_geo_package(orthophoto.id, file.id, _preload_content=False)
 
-                    #Writes the geopackage to file
+                    #Writes the geopackage to file in chunks
                     with open(f'rapid_orthophotos/{orthophoto.name}_{orthophoto.id}.gpkg', 'wb') as outfile:
-                        outfile.write(geo_package)
-                except ApiException as e:
-                    print(f"For rapid orthophoto {orthophoto.name}, {orthophoto.id}. Exception when calling RapidOrthophotoFilesApi->get_rapid_orthophoto_geo_package: %s\n" % e)
+                      for chunk in response.stream(65536):
+                        outfile.write(chunk)
+
+                except Exception as e:
+                    print(f"For rapid orthophoto {orthophoto.name}, {orthophoto.id}. Exception when downloading orthophoto: %s\n" % e)
+                finally:
+                    if (response != None):
+                      try:
+                        response.drain_conn()
```

Comment:
This is not necessary if all data has been read, so it could be moved to exception handler, but left it here for simplicity.

Releasing connection after finished operation, according to
https://urllib3.readthedocs.io/en/stable/advanced-usage.html#streaming-and-i-o